service/k8sclient/k8sclient.go

```diff
@@ -84,6 +87,19 @@ const (
 	psmdbAPIVersion        = "psmdb.percona.com/v1-6-0"
 	psmdbSecretNameTmpl    = "dbaas-%s-psmdb-secrets"
 	defaultPSMDBSecretName = "my-cluster-name-secrets"
+
+	// Max size of volume for AWS Elastic Block Storage service is 16TiB.
+	maxVolumeSizeEBS uint64 = 16 * 1024 * 1024 * 1024 * 1024
+)
+
+type kubernetesClusterType uint8
+
+const (
+	clusterTypeUnknown kubernetesClusterType = iota
```

Comment:
I would set values manually instead of `iota`, but it is a personal preference. 
If by accident the order of the contants change, it will affect the behavior.

Reply:
I am not sure what you mean. As long as one does not reference them using hardcoded values but uses the constant name, behavior imho should not change with constant order change, right?